security-policies/.regal/config.yaml

```diff
@@ -0,0 +1,35 @@
+rules:
+  idiomatic:
+    use-some-for-output-vars:
+      level: ignore
+  imports:
+    prefer-package-imports:
+      level: ignore
+  style:
+    avoid-get-and-list-prefix:
+      level: ignore
+    default-over-else:
+      level: ignore
+    external-reference:
+      level: ignore
+    line-length:
+      level: ignore
+    prefer-some-in-iteration:
+      level: ignore
+    prefer-snake-case:
+      level: ignore
+    rule-length:
+      level: error
+      max-rule-length: 50
+    todo-comment:
+      level: ignore
+    use-assignment-operator:
+      level: ignore
+  testing:
+    file-missing-test-suffix:
+      level: ignore
+    print-or-trace-call:
+      # unsure if these are intentional
```

Comment:
Actually, it's just that one function that uses those and I don't see how it is used anywhere. We can add `ignore:` comments for those uses and make a mental note to investigate if we even need that function.